### PR TITLE
aqemu: add platform specific code to get memory size

### PR DESCRIPTION
**Pull request by TURX**

### CODE REVIEW

File: app-emulation/aqemu/patches/aqemu.patchset [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
++#if defined Q_OS_LINUX || defined Q_OS_HAIKU
+ 
+ bool System_Info::Update_Host_USB()
+ {
+@@ -2503,7 +2503,7 @@ bool System_Info::Scan_USB_Proc( QList<VM_USB> &list )
+ 	return true;
+ }
+ 
+-#include <sys/sysinfo.h>
++//#include <sys/sysinfo.h>
```
Comment (korli) [inline app-emulation/aqemu/patches/aqemu.patchset]: #ifndef Q_OS_HAIKU

File: app-emulation/aqemu/patches/aqemu.patchset [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+--- a/src/System_Info.h
++++ b/src/System_Info.h
+@@ -26,6 +26,11 @@
+ #include "Utils.h"
+ #include "VM_Devices.h"
+ 
++#ifdef Q_OS_HAIKU
++// for memory size detection
++#include "OS.h"
++#endif
```
Comment (korli) [inline app-emulation/aqemu/patches/aqemu.patchset]: this should be in System_Info.cpp
should be #include <OS.h>

**Comment by korli** [comment]
Could you replace the old patchset?

**Comment by korli** [comment]
@scottmc  was this GCI Task already implemented?

**Comment by Begasus** [comment]
The patchset doesn't apply, looks like the patch to main.cpp is done on a git checkout?
https://build.haiku-os.org/buildmaster/master/x86_gcc2/logviewer.html?buildruns/3827/builds/34361.log

**Comment by korli** [comment]
Yeah, this needs a more careful review, sorry. I'm wondering whether the old patchset hadn't already code for to get memory size.

**Pull request merged**